packages/components/timepicker/src/components/osds-timepicker/osds-timepicker.scss

```diff
@@ -1,3 +1,15 @@
-:host {
-  font-weight: normal;
+.osds-timepicker {
+  :host(&#{&}--inline) {
+    display: inline-flex;
+
+    & .osds-select {
```

Comment:
we go with BEM on ODS, this should be more like:
```
:host(.osds-timepicker) {
  ...host only code
}

.osds-timepicker {
  &__timezones {
    ... select code
  }
}
```

Reply:
Class has been changed